### ISSUE TITLE
Suppressing Inspections

### ISSUE THREAD

**Issue by ItsEmpa** [dkandalov/live-plugin#190]
Is it possible to suppress an already existing inspection? Similar to how you can register new inspections and register problems with it, I would like to suppress warnings for arguments inside functions when the function has a specific annotation

**Comment by dkandalov** [comment]
Yes, something along the lines of this gist https://gist.github.com/dkandalov/590bdf339efe643a124e43082bed22d1
